test/test_workflows.py

```diff
@@ -63,7 +63,7 @@ def test_workflow_for_naming_schemes(self):
         folders.sort()
 
         # for debugging
-        # folders = folders[-1:]
+        folders = folders[-1:]
```

Comment:
is this intended?

Reply:
oh, this is just for debugging, good catch